.ai_lua/Instructions/toc_Format.instructions.md

```
@@ -0,0 +1,166 @@
+---
+applyTo: '**/*.toc'
+---
+
+# World of Warcraft Addon `.toc` File Specification (Definitive Guide)
+
+## 1. Core Concepts & Structure
+
+A `.toc` (Table of Contents) file is the mandatory entry point for any World of Warcraft addon. It acts as a manifest, defining the addon's identity and instructing the game client on which files to load.
+
+#### Key Structural Rules:
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)